### PR TITLE
ON-10219 # add hideFormSubmissionActions prop to OneBlinkForm

### PR DESCRIPTION
**Pull request by divporter**

### CODE REVIEW

**Comment by mymattcarroll** [comment]
This implementation could use a little bit more I think. The issue is that the onSubmit and onCancel props are still required based on the typescript.

I think we want to try and create a common component that can be used by the `<OneBlinkForm />` component as well as a new component: `<OneBlinkFormReadOnly />`. Then this component could force all elements in the form to be readonly and not require any of the submit/cancel props. Thoughts?

**Comment by divporter** [comment]
Mmm yeah, I hadn't considered the typescript part.  Yeah I considered a new component, but I thought we were against the idea of exporting another component, but I agree its much much cleaner.

I'm happy to fix it.

**Comment by mymattcarroll** [comment]
Looks good, just need to document the new component.

File: docs/OneBlinkReadOnlyForm.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# OneBlink Apps - ReactJS | Usage
+
+## `<OneBlinkReadOnlyForm />`
+
+Component for rendering a OneBlink Form in read-only mode. This component will render the form with all inputs disabled but will **not** render the the submit, cancel and save draft buttons.
+
+It is also recommended to import the `css` from this library as well.
+
+```js
+import { OneBlinkForm } from '@oneblink/apps-react'
```
Comment (mymattcarroll) [inline docs/OneBlinkReadOnlyForm.md:10]:
```suggestion
import { OneBlinkReadOnlyForm } from '@oneblink/apps-react'
```

File: docs/OneBlinkReadOnlyForm.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+function FormContainer() {
+  const isMounted = useIsMounted()
+
+  const handleFormError = React.useCallback(() => {
+    // handle form rendering error caused by a misconfigured form here...
+  }, [isMounted])
+
+  return (
+    <OneBlinkForm
+      captchaSiteKey={captchaSiteKey}
```
Comment (mymattcarroll) [inline docs/OneBlinkReadOnlyForm.md:72]: Dhould this be here

File: docs/OneBlinkReadOnlyForm.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  FormTypes,
+} from '@oneblink/apps'
+import {
+  IsOfflineContextProvider,
+  OneBlinkForm,
+  useIsMounted,
+} from '@oneblink/apps-react'
+import '@oneblink/apps-react/dist/styles.css'
+
+const captchaSiteKey = 'ENTER_YOUR_SITE_KEY_HERE'
```
Comment (mymattcarroll) [inline docs/OneBlinkReadOnlyForm.md:42]: and this?

File: docs/OneBlinkReadOnlyForm.md (hunk 2 above)
Comment (divporter) [reply docs/OneBlinkReadOnlyForm.md:72]: dang no

File: docs/OneBlinkReadOnlyForm.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+import ReactDOM from 'react-dom'
+import {
+  OneBlinkAppsError,
+  draftService,
+  submissionService,
+  FormTypes,
+} from '@oneblink/apps'
+import {
+  IsOfflineContextProvider,
+  OneBlinkForm,
```
Comment (mymattcarroll) [inline docs/OneBlinkReadOnlyForm.md:37]:
```suggestion
  OneBlinkReadOnlyForm,
```

File: docs/OneBlinkReadOnlyForm.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+function FormContainer() {
+  const isMounted = useIsMounted()
+
+  const handleFormError = React.useCallback(() => {
+    // handle form rendering error caused by a misconfigured form here...
+  }, [isMounted])
+
+  return (
+    <OneBlinkForm
```
Comment (mymattcarroll) [inline docs/OneBlinkReadOnlyForm.md:71]:
```suggestion
    < OneBlinkReadOnlyForm
```

**Pull request merged**